model_compression_toolkit/core/common/graph/base_node.py

```diff
@@ -553,7 +553,7 @@ def get_qco(self, tpc: TargetPlatformCapabilities) -> QuantizationConfigOptions:
         # Extract qco with is_match_type to overcome mismatch of function types in TF 2.15
         matching_qcos = [_qco for _type, _qco in tpc.layer2qco.items() if self.is_match_type(_type)]
         if matching_qcos:
-            if len(matching_qcos) > 1:
+            if len(matching_qcos) > 1 and matching_qcos[0] != tpc.tp_model.default_qco:
```

Comment:
I don't think that this solves the issue correctly.
I suspect that the case generating this issue is when there is a pytorch function that has two aliases, and that one of them appears in a TPC operation set and the other is not, thus returning the default qco.

Please try to reproduce the issue, identify the pytorch layer that has duplicate configs and let's think of a more robust solution.